conveyor :
    max_effort : 90
    id : 6
    id_list : [12,13]
    publish_frequency : 2
